app/scripts/components/boilerplate/header.jsx

```diff
@@ -10,8 +10,9 @@ const Header = React.createClass({
     return (
       <div className="header">
         <div className="logo">
-          <h1><T k="site-name" /></h1>
+          <img src="images/tz-flag.png"/>
```

Comment:
Not adding alt tag for now since we're setting up translation for it. Here is the ticket for the task: https://jira.dgfoundation.org/browse/TSD-227